src/app/planmylandoperation/sidebar/soils/soils.component.html

```diff
@@ -9,10 +9,17 @@
         <mat-icon class="smallIcon right">help</mat-icon>
         <br>
         <div class="row prl-10">
-            <mat-checkbox (change)="soilsVisibleChanged($event.checked)">Visible</mat-checkbox>
+            <label class="inline-flex">
```

Comment:
Traditional angular's way to setting checkbox as checked doesnot update the actual DOM 'checked' attribute.
This prevents us from doing css select such as `checkbox:checked` so I had to add a two way data binding using `FormsModule` and update the checked attribute to be later used in css selector as `checkbox[checked='true']`